src/styles/card.scss

```diff
@@ -0,0 +1,36 @@
+@import './stars';
+@import './colors';
+
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), all properties within a block should be alphabetized. Please rearrange the properties in `.card` to follow this rule.